student/src/main/java/tds/student/web/handlers/DialogFrameHandler.java

```diff
@@ -91,7 +92,7 @@ public String getDialogFrameContent () {
     {
       final AccLookup languageAccommodation = new AccLookup();
       languageAccommodation.add ("Language", contentLanguage);
-      itsDocument = contentRepository.findItemDocument(studentHelpFile, languageAccommodation, Server.getContextPath());
+      itsDocument = contentRepository.findItemDocument(studentHelpFile, languageAccommodation, Server.getContextPath(), new BrowserParser().isSupportsOggAudio());
```

Comment:
I think this handler should be using the `ContentService` and not the repository.  I know you're carrying it forward, but I personally prefer handlers to only talk to services especially since transaction boundaries are added on service methods.